src/environments/environment.model.ts

```diff
@@ -70,4 +72,7 @@ export interface Environment {
 
   // configuration of the available locales - hard coded for now
   locales: { lang: string; currency: string; value: string; displayName: string; displayLong: string }[];
+
+  // themes
+  themes: Theme[];
```

Comment:
I'd rather not see the `Theme` model being used here. We're not doing it with `Locale` either...

Reply:
decision for typed environments-model